.gitignore

```diff
@@ -0,0 +1,19 @@
+﻿*.exe
+*.dll
```

Comment:
please remove this file

Reply:
fix in bacad15,
btw i keep some necessary ignore files/dirs (e.g. auto_round.egg-info/)